pytorch_toolkit/ote/ote/metrics/classification/reid.py

```diff
@@ -61,8 +62,8 @@ def get_topk_dict(value, k=1):
         subprocess.run(
             f'python {REID_TOOLS}/main.py'
             f' --config-file {config_path}'
-            f' test.evaluate True'
             f' {update_config}'
+            f' test.evaluate True'
```

Comment:
what is a reason of this change?

Reply:
Before this PR `update_config` has contained the remaining rags only, but now it has a regular args also, so they should go first.